Hush/ProjectSettings/TagManager.asset

```diff
@@ -27,8 +28,8 @@ TagManager:
   - Interactible
   - Player
   - Enemy
-  - 
-  - 
+  - Door
+  - Vision
```

Comment:
What's the vision layer used for ?

Reply:
just for the enemy vision collider, since it was triggering the OnTriggerStay of the secret passage, i didnt want that collision so i put it on its own layer that doesn't collide with the default layer